src/containers/Tenant/Diagnostics/TopQueries/columns/columns.tsx

```diff
@@ -90,39 +91,38 @@ const oneLineQueryTextColumn: Column<KeyValueRow> = {
 };
 
 const queryHashColumn: Column<KeyValueRow> = {
-    name: TOP_QUERIES_COLUMNS_IDS.QueryHash,
-    header: TOP_QUERIES_COLUMNS_TITLES.QueryHash,
+    name: QUERIES_COLUMNS_IDS.QueryHash,
+    header: QUERIES_COLUMNS_TITLES.QueryHash,
     render: ({row}) => generateHash(String(row.QueryText)),
     width: 130,
 };
 
 const durationColumn: Column<KeyValueRow> = {
-    name: TOP_QUERIES_COLUMNS_IDS.Duration,
-    header: TOP_QUERIES_COLUMNS_TITLES.Duration,
+    name: QUERIES_COLUMNS_IDS.Duration,
+    header: QUERIES_COLUMNS_TITLES.Duration,
     render: ({row}) => formatToMs(parseUsToMs(row.Duration ?? undefined)),
     align: DataTable.RIGHT,
     width: 150,
 };
 
 const queryStartColumn: Column<KeyValueRow> = {
-    name: TOP_QUERIES_COLUMNS_IDS.QueryStartAt,
-    header: TOP_QUERIES_COLUMNS_TITLES.QueryStartAt,
+    name: QUERIES_COLUMNS_IDS.QueryStartAt,
+    header: QUERIES_COLUMNS_TITLES.QueryStartAt,
     render: ({row}) => formatDateTime(new Date(row.QueryStartAt as string).getTime()),
     resizeable: false,
-    defaultOrder: DataTable.DESCENDING,
 };
 
 const requestUnitsColumn: Column<KeyValueRow> = {
-    name: TOP_QUERIES_COLUMNS_IDS.RequestUnits,
-    header: TOP_QUERIES_COLUMNS_TITLES.RequestUnits,
+    name: QUERIES_COLUMNS_IDS.RequestUnits,
+    header: QUERIES_COLUMNS_TITLES.RequestUnits,
     render: ({row}) => formatNumber(row.RequestUnits),
     align: DataTable.RIGHT,
     width: 150,
```

Comment:
Please, set some initial width to running queries columns, this columns jump a little on sorting